charts/kusion/README.md

```diff
@@ -1,6 +1,6 @@
 # Kusion Chart
 
-![Version: 0.14.0](https://img.shields.io/badge/Version-0.14.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.14.0](https://img.shields.io/badge/AppVersion-0.14.0-informational?style=flat-square)  [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/kusion)](https://artifacthub.io/packages/helm/kusionstack/kusion)
```

Comment:
The `AppVersion` badge should be updated to match the new `Version` (0.14.2) for consistency. Currently, it's showing 0.14.0 which might cause confusion about the actual application version.